coulomb-tests/shared/src/test/scala/coulomb/Quantity.scala

```diff
@@ -330,7 +330,7 @@ object QuantityTests extends TestSuite {
         1.withUnit[Meter].show == ("1 m"),
         1.withUnit[Kilo %* Meter].show == ("1 km"),
         (1.5.withUnit[Meter] / 1.0.withUnit[Second]).show == ("1.5 m/s"),
-        1.0.withUnit[Second].pow[-1].show == ("1.0 s^(-1)"),
+        1.0.withUnit[Second].pow[-1].show == ("1.0 s^(-1)") || 1.0.withUnit[Second].pow[-1].show == ("1 s^(-1)"),
```

Comment:
scala.js mentions on the docs that a float `1.0` is printed as `1` without the trailing 0 unlike the jvm. Eventually `show` could take a formatter string but that maybe better dealt separately

Reply:
This is annoying, but I'm OK with leaving it this way since the number of tests affected won't be very large

Reply:
As a reference:
http://www.scala-js.org/doc/semantics.html